internal/db/changelog.go

```diff
@@ -41,3 +43,50 @@ func  (c *Client) GetAllChangelogEntries() ([]models.ChangelogEntry, error) {
 	return rules, nil
 }
 
+
+
+func (c *Client) GetChangeLogEntries(id int, limit *int) ([]models.BounceRule, error) {
+
+	stmt, err := c.Conn.Prepare("SELECT * From changelog WHERE rule_id = ?  ORDER BY created_at DESC LIMIT ?")

```

Comment:
Why use a prepare statement and not just a query? Prepared statements are typically meant to be used repeatedly, which isn't the case here.